css/css-flexbox-1/negative-margins-001.html

```diff
@@ -34,7 +32,7 @@
     }
 </style>
 <body>
-    <p>You should see a green rectangle with a black border, 60px wide. You should see no red.</p>
+    <p>You should see a green rectangle with a black border, 40px wide. You should see no red.</p>
```

Comment:
Why is this change better / more correct? Can we discuss this a bit more?

So far as I can tell, Chrome makes computes the width at 60px, Firefox at 50px (?), and I can't test Edge right now -- what does it do?

Reply:
@gsnedders It's now been close to 6 months - I'm no gitfu expert so if you know of a way to land this without this change and maybe spin up a separate issue let's land the rest.

Reply:
@gregwhitworth done (through interactive rebase and editing the commit by checking out the parent commit versions of the files)

Reply:
I've pushed this as 50a4c3652e14959e996fde52ae6c91437354552c on `gwhit-flex-changes-2`, will open PR for it soon.

Reply:
This change went into https://github.com/web-platform-tests/wpt/pull/7947 which has been merged.